### DISCUSSION TITLE
Mod compat for Sawmill

### DISCUSSION THREAD

**Discussion by saltyseadoggo** [TechReborn/TechReborn#2971] [Ideas]
The Sawmill's utility in modpacks- I'm currently playing All of Fabric 5- is severely harmed because it can only process vanilla wood types and rubber wood, which, in a large modpack, make up the minority of wood types. I would like to see it be able to handle other mods' wood types as well.
I would imagine this is doable with datagen at runtime? If it could grab every modded plank type with a corresponding log, stairs, slab and pressure plate, the mod could generate all of the needed recipes at runtime.

**Comment by Ayutac** [comment]
Give a list with mods with the woods pls

**Reply by ghost** [reply]
https://github.com/TeamAOF/All-of-Fabric-5/issues/289#issuecomment-1143679474
